packages/WillowCreekApp/jest.setup.js

```diff
@@ -22,6 +22,14 @@ jest.mock('react-navigation', () => {
   };
 });
 
+jest.mock('@react-navigation/native', () => {
+  const ActualNavigation = require.requireActual('@react-navigation/native');
+  return {
+    ...ActualNavigation,
+    SafeAreaView: require.requireActual('SafeAreaView'),
+  };
+});
+
```

Comment:
Can you help me understand the difference between this and the react-navigation mock (directly above)? I think my snaps aren't updating because of this and I'm having a hard time understanding the error.

Reply:
@vinnyjth 😃

Reply:
I added `@react-navigation/.*` to `transformIgnorePatterns` in `package.json jest` config which seems to have worked to resolve the error I was getting but now I'm getting